docs/_plugins/basemap-formats/allmaps-leaflet.md

```diff
@@ -0,0 +1,12 @@
+---
+name: allmaps-leaflet
+category: basemap-formats
```

Comment:
Please use the category "Non-map base layers"

Reply:
I consider this, but I think it's better placed in basemap-formats. Plugins in the 'Non-map base layers' like 'Leaflet-IIIF' show a (IIIF) image in ad-hoc pixel coordinates, whereas this plugin actually transforms/warps the original image to show it on the map in the right geographic location. In this way it's closer to the various W(T)MS plugins in the basemap-formats folder. See this [example](https://observablehq.com/@allmaps/leaflet-plugin). Let me know if you think the same.